src/components/browse/components/FacetList/FacetTermsList.js

```diff
@@ -351,8 +402,26 @@ const ListOfTerms = React.memo(function ListOfTerms(props){
         "key" : "facetlist"
     };
 
-    if (Array.isArray(collapsibleTerms)){
+    // show simple text input for basic search (search within returned values)
+    // or show SAYT control if item search is available
+    let facetSearch = null;
+    if (searchType === 'basic') {
+        facetSearch = (
+            <div className="text-small p-2">
+                <input className="form-control" autoComplete="off" type="search" placeholder="Search"
+                    name="q" onChange={onBasicTermSearch} key="facet-search-input" />
+            </div>);
+    } else if ((searchType === 'sayt') || (searchType === 'sayt_without_terms')) {
+        let { sayt_item_type: itemType = '' } = facet || {};
+        itemType = typeof itemType === 'string' && (itemType.length > 0) ? itemType : 'Item';
+        const baseHref = "/search/?type=" + itemType;
+        facetSearch = (
+            <div className="d-flex flex-wrap text-small p-2">
+                <SearchAsYouTypeAjax baseHref={baseHref} showTips={true} onChange={onSaytTermSearch} key={itemType} />
```

Comment:
Some styling notes for the dropdown: 
- I think the button would look nicer if it were bootstrap size "sm" rather than standard. Strikes me as a little large, currently. I don't remember if SAYTAJAX has a functioning size property, but would be a quick add.
- Or, if you want to keep the current button size, it might be worth fiddling with the styling to see if you can get the left and right margins to match up with the `.facet-list-element` (blue selection indicator thing). I think that might help, too.
- Range facet styling is also changing a bit, but the buttons used for those dropdowns are currently "btn-outline-dark" rather than "btn-outline-secondary." I'd normally ask you to switch to "btn-outline-dark" for the sake of consistency, but I do actually really like the way "btn-outline-secondary" looks here. @alexkb0009 Let me know if you have a preference between the two, too. If you also like "outline-secondary", I'll see if I can switch range facet to use that style in my current PR.

Reply:
Let's consider the style to converge to at the UI meeting. One consideration is how it looks from 4DN-side as well... though 4DN generally has "lighter" lines so pre-emptively I lean toward "btn-outline-secondary" a little bit as well due to that..